heatmap/src/heatmap.rs

```diff
@@ -196,40 +209,52 @@ impl Heatmap {
         self.summary.buckets()
     }
 
+    /// Returns the number of valid and active `Histogram` slices in the `Heatmap`
+    pub fn active_slices(&self) -> usize {
+        if Instant::now() - self.first_tick > self.span {
+            self.slices()
+        } else {
+            self.idx.load(Ordering::Relaxed) + 1
+        }
+    }
+
+    /// Returns the DateTime representation of when the `HeatMapi` was created
```

Comment:
Typo - `s/HeatMapi/Heatmap/`

Reply:
Also should put `DateTime` in single backticks

Reply:
fixed